packages/react-components/src/components/ParticipantsButton.tsx

```diff
@@ -134,27 +135,25 @@ export const ParticipantsButton = (props: ParticipantsButtonProps): JSX.Element
 
   const defaultMenuProps = useMemo((): IContextualMenuProps => {
     const menuProps: IContextualMenuProps = {
+      title: 'In this call',
+      styles: participantsButtonMenuPropsStyle as IContextualMenuStyles,
```

Comment:
Instead of typecasting `participantsButtonMenuPropsStyle` here as `IContextualMenuStyles`
I would recommend creating this style with a type of `IContextualMenuStyles` in the styles file so that in future anyone modifying with the styles will get proper typehints